uber/site_sections/registration.py

```diff
@@ -564,7 +567,7 @@ def new(self, session, show_all='', message='', checked_in=''):
         if show_all:
             restrict_to = [Attendee.paid == c.NOT_PAID, Attendee.placeholder == False]
         else:
-            restrict_to = [Attendee.registered > datetime.now(UTC) - timedelta(minutes=90)]
+            restrict_to = [Attendee.paid != c.NEED_NOT_PAY, Attendee.registered > datetime.now(UTC) - timedelta(minutes=90)]
```

Comment:
Although we'll probably stop using this page once badges becomes free, it struck me that there's never any reason for a need not pay badge to ever show up here, so I added that filter.  Now free badges added by admins on-site won't show up here either.